yb-voyager/cmd/importData.go

```diff
@@ -839,15 +840,16 @@ func doOneImport(task *SplitFileImportTask, targetChan chan *tgtdb.Target) {
 
 			dbVersion := targetServer.DB().GetVersion()
 
-			for i, statement := range IMPORT_SESSION_SETTERS {
-				if checkSessionVariableSupported(i, dbVersion) {
-					_, err := conn.Exec(context.Background(), statement)
-					if err != nil {
-						utils.ErrExit("import file %q: run query %q on %q: %s", inProgressFilePath, statement, targetServer.Host, err)
+			if !addCustomSessionVars(conn) {
```

Comment:
I will prefer to have both `if` and `else` condition here so make the code more readable instead of doing it secretly in one function.

1. IF condition: check `utils.FileOrFolderExists(customFilePath)` if true then call `useCustomSessionVars` fuction
2. Else condition: current behaviour.